.pre-commit-config.yaml

```
@@ -1,7 +1,9 @@
+autoupdate_schedule: 'monthly'
```

```suggestion
autoupdate_schedule: 'quartlery'
```

That should be enough. I'd like to have this project very low maintenance.